ethereum-consensus/src/clock.rs

```diff
@@ -12,6 +12,7 @@ use std::{
 pub const MAINNET_GENESIS_TIME: u64 = 1606824023;
 pub const SEPOLIA_GENESIS_TIME: u64 = 1655733600;
 pub const GOERLI_GENESIS_TIME: u64 = 1616508000;
+pub const HOLESKY_GENESIS_TIME: u64 = 1694786100;
```

Comment:
```suggestion
pub const HOLESKY_GENESIS_TIME: u64 = 1694786400;
```

genesis time is `MIN_GENESIS_TIME` and at least `GENESIS_DELAY`

bc of how we are starting the network, we know ahead of time what the genesis time will be

Reply:
can you please update the constant?